tests/wikidata/test_transform.py

```diff
@@ -62,9 +64,11 @@ def test_transform_wikidata_organization_to_organization(
         )
     )
 
-    assert len(extracted_organizations) == 1
+    if not extracted_organizations:
+        pytest.fail("No extracted organization.")
```

Comment:
```suggestion
    assert extracted_organizations
```

Reply:
updated in 108989566acf52f31953594e8d8cbd61be22be7a